provisioning/roles/chrome/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+- name: Download latest chrome
```

Comment:
Of course, I'd prefer chromium too. It just seems that Chromium is not something you download from a website, but get via your distro instead. And distroes like ubuntu and debian often have outdated packages... but maybe there is a special repo for that?